scripts/utils.py

```diff
@@ -88,10 +90,37 @@ def parse_sol_version(source):
     return DEFAULT_SOLC_VERSION
 
 
-def handle_process_output_and_exit(error):
-    """Processes stdout and stderr from a subprocess CalledProcessError."""
-    if error.output:
-        logging.info(error.output)
+def run_cmd(cmd, loglevel=logging.WARNING):
```

Comment:
`truffle compile` is fast enough in most cases for us to avoid this function, couldn't this function be replaced by `check_output` in this case?

For Securify, wouldn't it be better to modify the output of Securify directly?

Reply:
I meant that waiting for the completion of the `truffle` process is fine, so `check_output` from `subprocess` is good enough, no need to add more code than necessary, this can make it harder to maintain in the long run.

Reply:
Ohhh ok. But the `run_cmd()` function is defined anyways for Securify. But sure I can change that to `check_output()`.

Reply:
Not quite, the `check_output()` function waits for the process to finish to return. Therefore, you would not get live progress information from Securify, but only get it in bulk once Securify finished, which defeats the purpose of having progress information in the first place (to show that something is currently running and that the application is not frozen). However, the `run_cmd()` could be in inlined in `project.py` since it would only be used once there. Would you want me to inline it?

Reply:
Of course you cannot use `check_output()` to run Securify as, indeed, it would defeat the purpose of having progress information, but what about `run`?

Reply:
`run` indeed waits for the command to finish to return. Are you sure it does not transmit the output of the running command while waiting for its completion?

Reply:
You mean that it pipes the output directly to stdout from the current running python instance? Yes maybe there is a way to do this by passing something like `stdout=sys.stdout` in the run parameters. I will try it to see if I can make that work.

Reply:
Ok yes, I checked, this seems to work. I will implement the changes soon.